packages/@o3r/workspace/package.json

```diff
@@ -106,13 +107,13 @@
     "zone.js": "~0.14.2"
   },
   "generatorDependencies": {
-    "@angular/material": "~17.3.0",
-    "@ngrx/router-store": "~17.2.0",
-    "@ngrx/effects": "~17.2.0",
-    "@ngrx/store-devtools": "~17.2.0"
+    "@angular/material": "~18.0.0",
+    "@ngrx/router-store": "~18.0.0",
+    "@ngrx/effects": "~18.0.0",
+    "@ngrx/store-devtools": "~18.0.0"
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node": "^18.19.1 || ^20.11.1 || >=22.0.0"
```

Comment:
`^18.19.1 \|\| ^20.11.1 \|\| ^22.0.0`

Reply:
is that needed ?
we are using the same as in angular repo https://github.com/angular/angular/blob/main/packages/core/package.json#L8

Reply:
According to their docs: https://angular.dev/reference/versions#actively-supported-versions
But maybe for some reason they do not want to block future majors. So not blocking.

Reply:
Oh sorry, I didn't noticed the `^22` vs `>=22` I was too focused on the `\|\|` :D
I'm not sure why the mismatch, but that means we are pretending to support the odd versions of Node after 22
I suggest we keep it aligned with the package.json of angular for now but keep an eye on it for future releases of Node and Angular